app/src/main/java/zapsolutions/zap/connection/btcPay/BTCPayConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package zapsolutions.zap.connection.btcPay;
+
+public class BTCPayConfiguration {
```

Comment:
I only added the fields that we currently have in use.

Reply:
"certificateThumbprint" and "ssl" should maybe also included.
If I understood it correct, having ssl=false would mean falling back to certificateThumbprint, which actually is still ssl, as it uses LNDs generated sslCertificate instead of letsencryt one.
This case would have to be implemented in the other files as well though.
@rockstardev can you maybe comment on this? Did I understood that correct and is this case acutally important or does it not happen anyway?